From 838d06310d174cd98f12a4b2c56ed7ebfb7a3a28 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:08:43 -0400
Subject: [tools] perf: Prettify the tags/TAGS/cscope targets output

Message-id: <1410181752-25631-279-git-send-email-jolsa@redhat.com>
Patchwork-id: 91924
O-Subject: [PATCH RHEL7.1 BZ1134356 278/307] perf tools: Prettify the tags/TAGS/cscope targets output
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit b0815d07ec12959da2104822561e0df26dc4bf3d
Author: Jiri Olsa <jolsa@kernel.org>
Date: Wed Jun 4 14:43:58 2014 +0200

description
===========
Add tags/TAGS/cscope targets to the quiet family.

  $ make tags cscope
  BUILD:   Doing 'make -j4' parallel build
  GEN      tags

  $ make cscope
  BUILD:   Doing 'make -j4' parallel build
  GEN      cscope
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Makefile.perf b/tools/perf/Makefile.perf
index ae20edf..9670a16 100644
--- a/tools/perf/Makefile.perf
+++ b/tools/perf/Makefile.perf
@@ -819,15 +819,15 @@ TAG_FOLDERS= . ../lib/traceevent ../lib/api ../lib/symbol
 TAG_FILES= ../../include/uapi/linux/perf_event.h
 
 TAGS:
- $(RM) TAGS
+ $(QUIET_GEN)$(RM) TAGS; \
  $(FIND) $(TAG_FOLDERS) -name '*.[hcS]' -print | xargs etags -a $(TAG_FILES)
 
 tags:
- $(RM) tags
+ $(QUIET_GEN)$(RM) tags; \
  $(FIND) $(TAG_FOLDERS) -name '*.[hcS]' -print | xargs ctags -a $(TAG_FILES)
 
 cscope:
- $(RM) cscope*
+ $(QUIET_GEN)$(RM) cscope*; \
  $(FIND) $(TAG_FOLDERS) -name '*.[hcS]' -print | xargs cscope -b $(TAG_FILES)
 
 ### Detect prefix changes
-- 
1.7.1